sktime/regression/compose/_ensemble.py

```diff
@@ -411,28 +299,55 @@ def predict(self, X):
 
         # Parallel loop
         y_hat = Parallel(n_jobs=n_jobs, verbose=self.verbose)(
-            delayed(e.predict)(X) for e in self.estimators_)
+            delayed(e.predict)(X, check_input=True) for e in self.estimators_)
 
         return np.sum(y_hat, axis=0) / len(self.estimators_)
 
-    def _validate_X_predict(self, X):
-        n_features = X.shape[1] if X.ndim == 2 else 1
-        if self.n_features_ != n_features:
-            raise ValueError("Number of features of the model must "
-                             "match the input. Model n_features is %s and "
-                             "input n_features is %s "
-                             % (self.n_features_, n_features))
-        return X
+    def _set_oob_score(self, X, y):
+        """
+        Compute out-of-bag scores."""
+        check_X_y(X, y)
+        check_X(X, enforce_univariate=True)
+        X = check_array(X, dtype=DTYPE, accept_sparse='csr')
```

Comment:
This doesn't look right, probably `check_X_y(X, y, enforce_univariate=True)`

Reply:
Fixed